README.md

```diff
@@ -33,6 +36,133 @@ poetry install
 
 ---
 
+## 🧪 CLI Usage
+
+After installing, the CLI provides a single command:
+
+```bash
+chaci integrate-rock METADATA_FILE BASE_BRANCH ROCK_IMAGE [GITHUB_TOKEN] [GITHUB_USERNAME] [--clone-dir PATH]
```

Comment:
suggestion(blocking): make the optional arguments with flags rather than positional. The optional positional arguments can lead to ambiguous parsing.

Reply:
That makes perfect sense ... added